rustls-libssl/tests/config.c

```diff
@@ -14,20 +14,38 @@ static const int conf_flags[] = {SSL_CONF_FLAG_SERVER, SSL_CONF_FLAG_CLIENT,
 
 #define NUM_FLAGS (sizeof(conf_flags) / sizeof(conf_flags[0]))
 
-static const char *supported_cmds[] = {
-    "-min_protocol", CUSTOM_PREFIX "min_protocol",
-    "MinProtocol",   CUSTOM_PREFIX "MinProtocol",
-
-    "-max_protocol", CUSTOM_PREFIX "max_protocol",
-    "MaxProtocol",   CUSTOM_PREFIX "MaxProtocol",
-
-    "VerifyMode",    CUSTOM_PREFIX "VerifyMode",
-
-    "-cert",         CUSTOM_PREFIX "cert",
-    "Certificate",   CUSTOM_PREFIX "Certificate",
-
-    "-key",          CUSTOM_PREFIX "key",
-    "PrivateKey",    CUSTOM_PREFIX "PrivateKey"};
+static const char *supported_cmds[] = {"-min_protocol",
```

Comment:
oof, i guess clang-format did this since clang-format --dry-run is happy with it?

Reply:
Yeah :-( It's quite ugly. For a little while it was formatting it _even worse_than this. I wasn't able to find a way to coerce it into something I'd consider "normal". I think we'll have to live with it or tweak the clang formatting rules (?).